numpy/core/numeric.py

```diff
@@ -2262,6 +2227,9 @@ def allclose(a, b, rtol=1.e-5, atol=1.e-8):
         The relative tolerance parameter (see Notes).
     atol : float
         The absolute tolerance parameter (see Notes).
+    equal_nan : bool
+        Whether to compare NaN's as equal.  If True, NaN's in `a` will be
+        considered equal to NaN's in `b` in the output array.
```

Comment:
Need a `.. versionadded:: 1.10` here, either before the first line (I'm tending that way), of after a blank line at the end.